Documentation/kubernetes-on-aws.md

```diff
@@ -49,13 +59,16 @@ mv ${PLATFORM}/kube-aws /usr/local/bin
 
 Configure your local workstation with AWS credentials using one of the following methods:
 
-### Method 1: Environment variables
+### Method 1: Configure command
 
-Set `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to the values of your AWS access and secret keys, respectively:
+Provide the values of your AWS access and secret keys, and optionally default region and output format:
 
 ```sh
-$ export AWS_ACCESS_KEY_ID=AKID1234567890
-$ export AWS_SECRET_ACCESS_KEY=MY-SECRET-KEY
+$ aws configure
```

Comment:
Now I have to have `aws` to use kube-aws; previously these instructions told me how to set env vars or write `~/.aws/credentials` by hand.